docs/development/code-conventions.rst

```diff
@@ -564,14 +496,14 @@ use `delegate_to`.
          key: "setup.run"
          state: "reset"
 
-Default EXAMPLE parameters
-^^^^^^^^^^^^^^^^^^^^^^^^^^
+Format common parameters consistently
```

Comment:
this header needs to be reverted. EXAMPLE is a literal block in the ansible module. So it is not sufficient to just say "format common parameters". We need to very specifically format the EXAMPLES parameter with the mentioned params.

Reply:
I am trying to make all the titles actions. Can I use something like "Set default EXAMPLE parameters"?

Reply:
@selhorn that's fine